.github/workflows/pr-cpu.yaml

```diff
@@ -36,6 +36,11 @@ jobs:
             markers: 'not daily and not remote and not gpu and not vision and not doctest'
             pytest_command: 'coverage run -m pytest'
             composer_package_name: 'composer'
+          - name: 'cpu-3.10-2.0-composer'
```

Comment:
can you update daily tests to include this please?

Reply:
added